utils/appliance/__init__.py

```diff
@@ -270,9 +270,13 @@ def __enter__(self):
 
     def _screenshot_capture_at_context_leave(self, exc_type, exc_val, exc_tb):
 
-        from fixtures.artifactor_plugin import fire_art_hook
-        from pytest import config
-        from fixture.pytest_store import store
+        try:
```

Comment:
I was thinking about it and it would be entirely possible that the fixture imports  could trigger something like that as well.